### PR TITLE
Feature/refactor-home-components

### PR DESCRIPTION
**Pull request by davidleach724**
# Description
This branch refactors the following components: App, Home, Login, NavBar, and StateList
​
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor Code
​
## How Has This Been Tested?
- [x] npm start
- [ ] console.log()
- [ ] dev tools
- [ ] cypress
​
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
​
## Additional Comments for Team Members if Applicable 
I did not see day.js in this change - hold off merging until we discuss that issue

### CODE REVIEW

File: src/Components/LogIn/LogIn.js [hunk 1]
```diff
@@ -1,46 +1,50 @@
...
-    <div className='login-container'>
-      <img src={plant2} alt='second plant sketch' className='single-plant'/>
-      <h3 className='select-name'>Select Your User Name</h3>
-      <select className='user-list' onChange={e => handleChange(e.target.value)}>
+    <div className="login-container">
+      <img src={ plant2 } alt="second plant sketch" className="single-plant" />
+      <h3 className="select-name">Select Your User Name</h3>
+      <select
+        className="user-list"
+        onChange={(e) => handleChange(e.target.value)}
```
Comment (hheyhhay) [inline src/Components/LogIn/LogIn.js]: I've been putting spaces between here: eg { (e) => handleChange(XXX) } 

We can do either but let's stay consistant :)

File: src/Components/LogIn/LogIn.js [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
+      >
         {!currentUserID && <option>Click to Expand...</option>}
         {userList}
       </select>
       <Link to={conditionalLink}>
-        <img src={loginImg} className='menu-btn-login' alt='log in btn'/>
-
+        <img src={loginImg} className="menu-btn-login" alt="log in btn" />
```
Comment (hheyhhay) [inline src/Components/LogIn/LogIn.js]: Same with here: 
src={ loginImg }

File: src/Components/StateList/StateList.js [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
+        { state }
+      </option>
+    );
+  });
 
   return (
-    <select className='input-signup' onChange={e => handleLocationChange(e.target.value)}>
+    <select
+      className="input-signup"
+      onChange={(e) => handleLocationChange(e.target.value)}
```
Comment (hheyhhay) [inline src/Components/StateList/StateList.js]: `onChange={ (e) => handleLocationChange(e.target.value) }`

**Review by hheyhhay — APPROVED** [review]

File: src/Components/LogIn/LogIn.js (hunk 2 above)
Comment (davidleach724) [reply src/Components/LogIn/LogIn.js]: Yeah cool that's what I did for the rest, I guess I missed this one

**Pull request merged**